test/fixtures.ts

```diff
@@ -465,6 +465,89 @@ export default {
           'e405eb422416ed785237e1406fec41485d6f7dd09809b73a44ba45c16d6616c6',
       },
     ],
+    childKeyDerivationFixture: {
+      mnemonic:
+        'kick find flash bar winner drastic flip half exotic outer salad future loud review palace',
```

Comment:
For now, I just used `kick find` mnemonic, which is not ideal but I randomly used it and then sticked with it. I plan to replace it with different one. Honestly, I spent quite some time looking for some official test vectors, no luck, I ll keep looking, alternatively I ll try to generate some using some well-used cardano lib, maybe cardano-serialization-lib

Reply:
I suppose I can use that one to create the test vectors in place of `kick find`